templates/releases/releases.html

```diff
@@ -18,43 +18,38 @@
+                {% set link = "/" ~ release.name ~ "/" ~ release.version ~ "/" ~ release.target_name ~ "/" -%}
+            {%- else -%}
+                {% set link = "/crate/" ~ release.name ~ "/" ~ release.version -%}
+            {%- endif -%}
+                <a href="{{ link | safe }}" class="release">
+                    <div class="pure-g">
+                        <div class="pure-u-1 pure-u-sm-6-24 pure-u-md-5-24 name">
+                            {{ release.name }}-{{ release.version }}
+                        </div>
 
-                    <li>
-                        <a href="{{ link | safe }}" class="release">
-                            <div class="pure-g">
-                                <div class="pure-u-1 pure-u-sm-6-24 pure-u-md-5-24 name">
-                                    {{ release.name }}-{{ release.version }}
-                                </div>
+                        <div class="pure-u-1 pure-u-sm-14-24 pure-u-md-16-24 description">
+                            {{ release.description }}
+                        </div>
 
-                                <div class="pure-u-1 pure-u-sm-14-24 pure-u-md-16-24 description">
-                                    {{ release.description }}
-                                </div>
-
-                                {% if release_type == 'owner' -%}
-                                    <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date"
-                                        title="Published {{ release.release_time | timeformat(relative=true) }}">
-                                        {{ release.stars }}
-                                        {{ "star" | fas }}
-                                    </div>
-                                {%- else -%}
-                                    <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date"
-                                        title="{{ release.release_time | date(format='%FT%TZ') }}">
-                                        {{ release.release_time | timeformat(relative=true) }}
-                                    </div>
-                                {%- endif %}
+                        {% if release_type == 'owner' -%}
+                            <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date"
+                                title="Published {{ release.release_time | timeformat(relative=true) }}">
+                                {{ release.stars }}
+                                {{ "star" | fas }}
+                            </div>
+                        {%- else -%}
+                            <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date"
+                                title="{{ release.release_time | date(format='%FT%TZ') }}">
+                                {{ release.release_time | timeformat(relative=true) }}
                             </div>
-                        </a>
-                    </li>
-                {%- endfor -%}
-            </ul>
```

Comment:
Why did you remove these HTML tags?

Reply:
We cannot focus on a `<li>`, so I removed the `<ul>` and moved the `<a>` out.